main/forms.py

```diff
@@ -0,0 +1,12 @@
+from allauth.account.forms import SignupForm
```

Comment:
This custom form tells the AllAuth backend what all fields it is supposed to expect from the signup form . I had to do this as by default it does not expect name from signup form . Refer comment on line 86 in settings.py